### DISCUSSION TITLE
Crosshair and tooltip in Data chart

### DISCUSSION THREAD

**Discussion by MiryamSubiza** [IgniteUI/igniteui-angular#13970] [Q&A]
I have enabled the crosshair layer in my chart (chart type "igx-data-chart" and series type "igx-scatter-line-series"), and the tooltip and the crosshair layer are not working as I expected when there is more than one series. 

I expected the tooltip to show all the existing "y" values for the same "x" value and the vertical crosshair to be only one line.
Instead, what the tooltip is showing is the closest point to the cursor of each series, and one vertical line for each series. 

In the following screenshots the cursor is placed in the red cross and, depending on the cursor location, the tooltip shows different values. I would expect to see in both cases the two values for the year 2015 in a single tooltip, and a vertical line where the 2015 value is.

![image](https://github.com/IgniteUI/igniteui-angular/assets/14543376/07215451-61b6-4b65-96ec-74cd742b7847)

![image](https://github.com/IgniteUI/igniteui-angular/assets/14543376/f3c8d871-53c1-47cf-b01a-ce4b4f36471d)

In [this example from the docs](https://www.infragistics.com/products/ignite-ui-angular/angular/components/charts/features/chart-axis-layouts#axis-locations-example) I see that the tooltip works as I would expect, but the chart type is "igx-category-chart":

![image](https://github.com/IgniteUI/igniteui-angular/assets/14543376/4356a2ef-f919-4ab6-863f-77c26c35d1b1)

In case a value was not available for a series I would expect the tooltip to hide the series in the tooltip or to show in the tooltip that the value for that series is not available.

**Comment by mddifilippo89** [comment]
Hello and thank you for reaching out. Are you using the DataTooltipLayer? If so, you can set the groupingMode to Grouped and it will join the tooltips as shown in the Category Chart.

**Reply by MiryamSubiza** [reply]
Hello, thank you for your response.
When I use the DataToolTipLayer and I set the groupingMode to "Grouped", then a single tooltip is shown.

The issue is that I expected to also have a single vertical crosshair layer, where all the "y" values for the same "x" value are displayed in the tooltip. However, there may be more than one vertical crosshair depending on the position of the cursor, that is to say, one vertical line for each series.

I created an example on the `feat/tooltip` branch of [this repository](https://github.com/remshams/igx-data-chart-crosshair)

You will see that depending on the cursor location there may be one or two vertical crosshairs, instead of always showing a single crosshair for each value in the "x" axis.

![image](https://github.com/IgniteUI/igniteui-angular/assets/14543376/fa87ae72-02a5-499a-9a28-8f2a916df157)
![image](https://github.com/IgniteUI/igniteui-angular/assets/14543376/f3c703ca-166b-471a-8d02-db5f4f1ec1bb)
